### DISCUSSION TITLE
livewire v3 (beta) hot reloading / autoreload / hmr

### DISCUSSION THREAD

**Discussion by camiant** [livewire/livewire#5925] [1. Help] [answered]
I'm searching around without any luck...

As of this article https://laravel-news.com/livewire-v3-features (_Hot reloading without a build step_)

... and according to this video https://www.youtube.com/live/f4QShF42c6E?feature=share&t=6586

... v3 should have hot-reloading like HMR in dev.

Is this feature not available yet, being in beta?

**Comment by joshhanley** [comment, marked as answer]
@camiant Hot reloading is built into the new Livewire dev tools that Caleb just demoed at Laracon. So you'll have access to it once it has been released 🙂

**Comment by camiant** [comment]
@joshhanley cool! thanks a lot for your contribution! 😊👍🏻

**Reply by joshhanley** [reply]
@camiant no worries! 🙂

**Reply by sbc640964** [reply]
Wasn't it published?
